### ISSUE TITLE
A.I refuses to build walls in 99% of the cases

### ISSUE THREAD

**Issue by Drfeelgoodsad** [Woprok/AOE4-AdvancedGameSettings#55]
Greetings! First i wanted to say thank you for this very awesome mod, i wish this would have been standard with the release of the Game.
Yet 1 year later we still are waiting :(

Anway, i just wanted to tell you that i stumbled over a problem with your Mod
Took me 100 hours to figure out why AI didn't wanted to place walls in the first place.
When i got the game at free2play week at steam i directly jumped into a session with your mod.

Well it doesn't matter if i use you mod activly as Game Mode, aslong this Mod is active, the A.I starts to refuse building Walls in 99% of the Cases, if i delete your Mod, they start to build again.

I can't really understand how the Mod is affecting the A.I in way like this o0? Did you changed maybe something? Is this intended? Or Is this a random conflict?

I just wanted to let you know, cause i really don't want to miss this mod :( been playing only Coop vs A.I  and because i usally need the Tuningpacks for other purpose (improving A.I as example or other great mods) i rely on your mod as Popcap and other settings.

Do you think there is any way to fix this?

Thanks for your time and this awesome mod <3

**Comment by Woprok** [comment]
TLDR: I didn't change anything to cause AI to behave even more stupid then normal... Basically any gamemode will make AI worse...

Explanation:
Any modded game causes AI to behave worse in certain way as it isn't coded to play with any change of rules.   
Singleplayer game cause AI to behave even worse due to reasons. This can be easily visible with playing Nomad in Singleplayer and Nomad in Multiplayer, where the AI will behave completely different even if everything is same...   

AI will build walls only if it feel threatened. This means that in most cases 1v1 Ai is much less likely to built walls. If you play in FFA setting with multiple AIs or multiple teams 3+ it might built walls more often as it will evaluate walls as more important.
